sentry-sample/src/main/java/io/sentry/sample/MyApplication.java

```diff
@@ -0,0 +1,17 @@
+package io.sentry.sample;
+
+import android.app.Application;
+
+// import io.sentry.android.core.SentryAndroid;
+
+/** Apps. main Application. */
+public class MyApplication extends Application {
+
+  @Override
+  public void onCreate() {
+    super.onCreate();
+
+    // how to init. Sentry manually
+    // SentryAndroid.init(this);
```

Comment:
The use-case of initializing manually is if users want to set stuff into `SentryOptions` like `BeforeSend` so if we go this route of having `SentryAndroid` static class with `init` at least we need a nice API like:

```java
SentryAndroid.init(o -> {
o.beforeSend = e -> { ... },
o.AndroidSepcificStuf = ...
});

```

Reply:
that's the idea, the one I've added is only if the user wants to init manually for some reason but still supporting our current set up with a minimal amount of code, of course this API could be extended.